docker/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:alpine as builder
+COPY . /go/src/lnaddrd
+WORKDIR /go/src/lnaddrd
+RUN go build
+
+FROM alpine
+MAINTAINER Frédéric Logier <fredix@protonmail.com>
+COPY --from=builder /go/src/lnaddrd/lnaddrd /usr/bin/
+RUN mkdir /usr/share/lnaddrd
+WORKDIR /usr/share/lnaddrd
+
+RUN apk add -U --no-cache ca-certificates
```

Comment:
What does this line do?

Reply:
the last line install ca-certificates from Mozilla. I need to connect to my lnd server which is using let's encrypt. But maybe it's not usefull if the user set the tls_cert_file. By the way, is there a way to not use tls_cert_file ? I'm using a reverse proxy (caddy) with let's encrypt so I start lnd with --no-rest-tls

Reply:
> By the way, is there a way to not use tls_cert_file ?

Unfortunately not, `tls_cert_file` is required for security. You shouldn't be relying on certificates issued by public certificate authorities when connecting to LND - You should be using the self-signed certificate created by LND. 

Otherwise, a compromised public certificate authority could be used to create an MitM attack on your node, intercepting your authentication credentials (macaroon), leading to a full LND compromise and loss of money.  

You might want to fix your LND setup to plug that vulnerability. This line should be removed unless there is some other need for it

Reply:
ok line removed